lib/gulp-index-file-stream.js

```diff
@@ -34,7 +34,10 @@ IndexFileStream.prototype._read = function() {
 
   this._indexStatic.generate(url, {
     meta: this._options.meta || {}
-  }, function(html) {
+  }, function(err, html) {
+    if (err) {
+      return this.emit('error', err);
+    }
```

Comment:
(commenting on this line instead since Github doesn't let me do it on [line 19](https://github.com/mozilla/teach.webmaker.org/pull/839/files#diff-5c5ee15f5e04d0241e2101cf5b6ac58aR19))

I'm just curious why we have to reverse the array.  Does it matter [which url gets popped first](https://github.com/mozilla/teach.webmaker.org/pull/839/files#diff-5c5ee15f5e04d0241e2101cf5b6ac58aR28)?